src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
-import React from 'react';
+import { UserInfo } from '../UserInfo';
+import commentsFromServer from '../../api/comments';
+import usersFromServer from '../../api/users';
+import { Post } from '../../types/post';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    userId,
+    title,
+    body,
+    id,
+  } = post;
+
+  const user = usersFromServer.find(curUser => curUser.id === userId);
+  const comments = commentsFromServer
```

Comment:
You do this logic every time when `PostInfo` rerenders. It's not okay from the performance side. Would be good to prepare the array once in `App.tsx` component and pass it to `<PostList />`